rt-raw-image.yml

```diff
@@ -4,14 +4,26 @@ components:
 
     DocumentImageResult:
```

Comment:
this is all wrong
this should be the same type for both RPRM_ResultType_RawUncroppedImage =16 and RPRM_ResultType_RawImage = 1 results.